resources/migrations/001_update_migrations.yaml

```diff
@@ -5317,6 +5317,38 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v49.2024-02-02T11:27:49
+      author: oisincoveney
+      comment: >-
+        Add report_card.initially_published_at
```

Comment:
any downsides to renaming this `published_at`? I'm just a boy with limited keystrokes and 120 characters of column width

Reply:
Ah okay, I didn't know that. `first_published_at` then?

Reply:
That was actually the name in the first iteration 😅 I'm not opposed to changing it but I think it's decent for now since we want to get this in